doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -3,351 +3,30 @@
-. Click *Assignments*.
-. In the *Data Outputs and Assignment* area, add a data output variable twice by using the same variable name.
-
-Expected result: Two variables with the same name are stored in the BPMN XML file.
-
-Actual result: The variables are not stored in the resulting BPMN XML file (XML) so when the process runs or loads, the data output assignments are missing.
-
-Workaround: None.
-
-== {PROCESS_ENGINE_CAP}
-.Task operations are slow [https://issues.jboss.org/browse/RHPAM-2259[RHPAM-2259]]
-
-Issue: Task operations such as *Start*, *Complete*, or *Claim* are noticeably slower when the {KIE_SERVER} API or `UserTaskService` is used. This issue appeared when a join was added to the `getTaskInstanceById` query. The performance of these operations will be restored to the previous state in a future patch release.
-
-
-endif::[]
-
-ifdef::PAM[]
-== {KIE_SERVER}
-.Query-related operations return errors when {KIE_SERVER} runs on either {JWS} or SpringBoot with DB2 [https://issues.jboss.org/browse/RHPAM-2292[RHPAM-2292]]
-
-Issue: Query-related operations return errors when {KIE_SERVER} runs on either {JWS} or SpringBoot with DB2.
-
-Steps to reproduce:
-
-. Create a `query-definitions.json` file that contains content similar to the following example and add it to {KIE_SERVER} on {JWS}:
-+
-[source]
-----
-
-[
-  {
-    "query-name" : "first-query",
-    "query-source" : "${org.kie.server.persistence.ds}",
-    "query-expression" : "select * from ProcessInstanceLog",
-    "query-target" : "PROCESS"
-
-  }
-]
-----
-. Start {KIE_SERVER}. {KIE_SERVER} tries to register the new default query.
-
-Expected result: The default query is successfully registered.
-
-Actual result: An error code is returned, for example:
-[source]
-----
-com.ibm.db2.jcc.am.XaException: [jcc][t4][10401][12066][4.24.92]
-XA exception: XAER_OUTSIDE ERRORCODE=-4228, SQLSTATE=null
-----
-
-Workaround: For {JWS}, set `defaultAutoCommit=false` in the `XADataSource`. For SpringBoot, set `spring.datasource.defaultAutoCommit=false` in application properties. This property must be added to the Narayana data source.
-endif::[]
-
-
 == {OPENSHIFT}
-.Validation sub-element is missing for data sources [https://issues.jboss.org/browse/RHPAM-2226[RHPAM-2226]]
+.In a high-availability authoring environment, when one user imports a project, another user is unable to see it [https://issues.jboss.org/browse/AF-2062[AF-2062]]
 
-Issue: If you create a custom template for deploying {KIE_SERVER} on {OPENSHIFT}
-and the server uses a MySQL or PostgreSQL database, {KIE_SERVER} data sources do not contain the validation sub-element where connection checkers and exception sorters are defined.
+Issue: When multiple users connect to a high-availability {CENTRAL} and one of them creates a project, another user can not see the project.
 
 Steps to reproduce:
+. Log in to {CENTRAL} as two different users (A and B) from two different hosts or browsers.
```

Comment:
The rendered output is not a bulleted list. ".." in a row repeated

Steps to reproduce: . Log in to Business Central as two different users (A and B) from two different hosts or browsers. . As user A and as user B, open the same space. . As user A, import a project using an external Git repository URL.

Reply:
Fixed, thanks.